src/components/Auth/InputField/Input/Input.tsx

```diff
@@ -0,0 +1,33 @@
+import { ChangeEvent, FocusEventHandler } from "react";
+import styles from "./Input.module.scss";
+
+interface Input {
+  className: string;
```

Comment:
Input 컴포넌트를 사용할때 항상 className을 주입해준다고 보기엔 어려울 것 같아요~
그래서 옵셔널하게 타입을 표현해줘야할 것 같습니다~
```suggestion
  className?: string;
```